config.toml

```diff
@@ -144,8 +144,8 @@ footer_about_disable = true
 [params.ui.feedback]
 enable = true
 # The responses that the user sees after clicking "yes" (the page was helpful) or "no" (the page was not helpful).
-no = 'Sorry to hear that. Please <a href="https://github.com/USERNAME/REPOSITORY/issues/new">tell us how we can improve</a>.'
-yes = 'Glad to hear it! Please <a href="https://github.com/USERNAME/REPOSITORY/issues/new">tell us how we can improve</a>.'
+no = 'Sorry to hear that. Please visit <a href="https://github.com/MIT-LCP/mimic-website/issues">to tell us how we can improve</a>.'
```

Comment:
Not tested, but this looks like it says "Please visit to tell us how we can improve"? Maybe change to something like "Please raise an issue to tell us how we can improve"?

Reply:
Thanks @tompollard!  I like your suggestion, I've adjusted it accordingly.